tests/integration/test_vulnerability_detector/test_scan_results/test_scan_providers_and_nvd_feeds.py

```diff
@@ -136,34 +147,32 @@ def check_vulnerability_event(package, cve):
     )
 
 
-def test_ubuntu_vulnerabilities_report(get_configuration, configure_environment, restart_modulesd,
-                                       mock_vulnerability_scan):
+def test_vulnerabilities_report(get_configuration, configure_environment, restart_modulesd,
+                                mock_vulnerability_scan):
     """
     Check if inserted vulnerable packages are reported by vulnerability detector
     """
-    nvd_vulnerabilities_number = len(nvd_vulnerabilities['vulnerabilities'])
-    provider_vulnerabilities_number = len(provider_vulnerabilities['vulnerabilities'])
+    # Check the vulnerabilities of packages inserted
+    for item in total_vulnerabilities:
+        check_vulnerability_event(item['package']['name'], item['cve']['cveid'])
```

Comment:
Why change the order? First the messages from:

```
A total of '{provider_vulnerabilities_number}' ...
```

and then

```
check_vulnerability_event(..)
```

Right?

Reply:
Because I am using a different message that appears after the vulnerabilities.